.github/scripts/changed-modules.sh

```
@@ -0,0 +1,84 @@
+# ROOT_DIR is the root directory of the repository.
+readonly ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)
+
+# define an array of modules that won't be included in the list
+readonly excluded_modules=("mockstorage")
+
+# modules is an array that will store the paths of all the modules in the repository.
+modules=()
+
+function is_excluded() {
+    for excluded_module in "${excluded_modules[@]}"; do
+        if [[ $1 == $excluded_module ]]; then
```

Shellcheck (0.10.0)

[warning] Quote the right-hand side of == in [[ ]] to prevent glob matching.

(SC2053)